src/test/kotlin/ua/marchenko/artauction/config/annotation/customScheduled/CustomScheduledAnnotationTest.kt

```diff
@@ -0,0 +1,117 @@
+package ua.marchenko.artauction.config.annotation.customScheduled
+
+import ch.qos.logback.classic.Logger
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.core.read.ListAppender
+import config.annotation.customScheduled.CustomScheduledTestServiceWithAnnotationImpl
+import config.annotation.customScheduled.CustomScheduledTestServiceWithoutAnnotationImpl
+import getRandomString
+import io.mockk.every
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.slot
+import io.mockk.verify
+import java.time.Duration
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.config.BeanDefinition
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory
+import org.springframework.context.ApplicationContext
+import org.springframework.context.event.ContextRefreshedEvent
+import ua.marchenko.artauction.common.annotation.scheduled.contextListener.CustomScheduledInvokerContextListener
+import java.util.concurrent.ScheduledExecutorService
+import java.util.concurrent.TimeUnit
+import kotlin.reflect.full.declaredMemberProperties
+import kotlin.reflect.jvm.isAccessible
+import kotlin.reflect.jvm.javaField
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.slf4j.LoggerFactory
+
+class CustomScheduledAnnotationTest {
```

Comment:
IMO, this test isn't helpful because you're mocking a ton of spring behavior. and if something is  changed in spring or your expectations from spring are wrong - this test will be false positive